README.md

```diff
@@ -11,11 +11,14 @@ In Xcode:
 1. In the build settings for the target that you're interested in
    dumping the AST for, set the `SWIFT_EXEC` user defined build setting
    to the path of `ast.py`.
+1. Add `AST_DUMP_FILE="$(SRCROOT)/$(TARGET_NAME).ast"` to 
+   `Preprocessor Macros`(`GCC_PREPROCESSOR_DEFINITIONS`) setting.
```

Comment:
Is this the right build setting for these? Surprised these get passed to swiftc 🤔

Reply:
Yeah. they still got passed to the `swiftc` - you can check by yourself - it's one minute to check, actually. I tried to find how we could prevent copy-pasting the output from Xcode, which is not working well for huge ASTs, and found that we can use preprocessor flags (which are still available for swiftc)

Reply:
Yea definitely, sounds good